src/components/Login.vue

```diff
@@ -16,8 +16,8 @@
 
                 <div v-if="tokenRequired">
                     <div class="form-floating mt-3">
-                        <input id="floatingToken" v-model="token" type="text" maxlength="6" class="form-control" placeholder="123456">
-                        <label for="floatingToken">{{ $t("Token") }}</label>
+                        <input id="otp" v-model="token" type="text" maxlength="6" class="form-control" placeholder="123456">
+                        <label for="otp">{{ $t("2FA Token") }}</label>
```

Comment:
`2FA Token` translation key doesn't exist in any language file. Please add it to en.js file

Reply:
Changing this key doesn't even make sense. Maybe we want to update the translation, in this case just update `en.js` instead.

Reply:
Reverted, I don't really care about that part